### ISSUE TITLE
Register in Symphony (Sellers)

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#54]
As a seller, I need to understand how to register in the OASIS+ procurement system (Symphony), so that when RFI/RFQs are posted, I am prepared to compete for solicitations.

- [Content](https://docs.google.com/document/d/1KibI_qCQFnc2qogOn7F4tIDeajtYcNrN2US73RmckRo/edit#heading=h.iygv7gu3pcbe)
- [Design](https://www.figma.com/proto/lWfZDygV7WbvGlea2PIrpa/OASIS-Playground?page-id=1448%3A4471&type=design&node-id=2547-8641&viewport=-4969%2C1003%2C0.24&t=CEMD1n5CrD5I1l7J-1&scaling=scale-down-width&starting-point-node-id=2111%3A11279)

**Status**

- [x]  Content created
- [x]  Content reviewed by SD
- [x]  Content reviewed by SME
- [x]  Design finalized
- [x]  Content finalized
- [ ]  Page developed

**Comment by klohman** [comment]
@ekidenda @PowderD This is ready for your review [here](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/mmeyer/seller-alt-updates/sellers-guide/register-in-symphony/)

Notes: 
The Symphony procurement tool link will not work (we'll need to go back and add before launch). 
The View all Symphony resources will give you a 404 page because we haven't built that page yet, but you can check the url path to confirm the page is correct (i.e. resources/)

**Comment by ekidenda** [comment]
The page looks good. Active links work well. 

Curious Question:
Is the intent for linked pages to open in a new tab? Currently, it doesn't and navigates users away from the web page. Maybe this is ideal for hand held devices...? Just thinking aloud and considering what's best for users.

**Comment by klohman** [comment]
> The page looks good. Active links work well.
> 
> Curious Question: Is the intent for linked pages to open in a new tab? Currently, it doesn't and navigates users away from the web page. Maybe this is ideal for hand held devices...? Just thinking aloud and considering what's best for users.

Yes, the intent is links will open in the same tab. There is [WCAG guidance](https://www.w3.org/TR/WCAG20-TECHS/G200.html) to avoid this, as well as [usability best practices](https://www.nngroup.com/articles/new-browser-windows-and-tabs/).

**Comment by vickimcfadden** [comment]
Thanks all. Moving this to done!
